constants/constants.go

```diff
@@ -95,8 +95,24 @@ const (
 	// MDLDataDir defines location for MDL data dir
 	MDLDataDir = "mdldata"
 
-	// SuperNodeDefaultLogLevel defines supernode log level
-	SuperNodeDefaultLogLevel = "debug"
+	// LogConfigDefaultCompress defines supernode log compress
+	LogConfigDefaultCompress = true
+	// LogConfigDefaultMaxSizeMB defines supernode log max size
+	LogConfigDefaultMaxSizeMB = 100
+	// LogConfigDefaultMaxAgeDays defines supernode log max age
+	LogConfigDefaultMaxAgeDays = 3
+	// LogConfigDefaultMaxBackups defines supernode log max backups
+	LogConfigDefaultMaxBackups = 10
+
+	// SuperNodeDefaultCommonLogLevel defines supernode common log level
+	SuperNodeDefaultCommonLogLevel = "debug"
```

Comment:
For user - then default log of common log should be `info`, and subsystem (p2p, metadb,...) - I think should be `error` to reduce log - could you confirm with ayk?
defaultCommonLogLevel    = "info"
defaultSubSystemLogLevel = "error" // disable almost logs

Reply:
@nguyen202 as discussed with ayk, I change common debug to `info` and other log-level to `error` only.